src/openrct2/object/ObjectJsonHelpers.h

```diff
@@ -29,6 +29,7 @@ namespace ObjectJsonHelpers
     float GetFloat(const json_t * obj, const std::string &name, const float &defaultValue = 0);
     std::vector<std::string> GetJsonStringArray(const json_t * arr);
     std::vector<int32_t> GetJsonIntegerArray(const json_t * arr);
+    std::vector<float_t> GetJsonRealArray(const json_t * arr);
```

Comment:
Please don't use floating point values for code that touches logic, different compilers and different optimisation flags will lead to divergent results.

Reply:
this should be resolved now